topUsersData.sh

```diff
@@ -0,0 +1,5 @@
+ssh -i /home/dashboard/VizDashKey root@crane-head > squeueDataRaw.csv 
+ssh -i /home/dashboard/VizDashKey root@tusker-head >> squeueDataRaw.csv
+
+python slurmToRuby.py > squeueData.csv
+rm squeueDataRaw.csv
```

Comment:
Man this is ugly.  Really, you have root ssh keys on viz to log into the head nodes?